### PR TITLE
chore: make the Kafka admin server URL configurable

### PR DESCRIPTION
**Pull request by b1zzu**
**Why**

When deploying the UI in a different environment from stage or production the Kafka admin server route and/or protocol can change and therefore we must be able to re-configure it in the UI

Blocked: https://github.com/redhat-developer/app-services-ui-shared/pull/117

### CODE REVIEW

**Review by riccardo-forina — APPROVED** [review]
LGTM

**Comment by pmuir** [comment]
Would it be better to use an actual string interpolation format here e.g. printf or a template?

**Comment by b1zzu** [comment]
> Would it be better to use an actual string interpolation format here e.g. printf or a template?

Didn't find anything easy to use in JavaScript without adding a new library, or writing a `.format()` method that can be copied and past from StackOverflow

File: src/app/pages/Kafka/kafka-instance.ts [hunk 1]
```diff
@@ -3,11 +3,21 @@ import { Configuration, DefaultApi, KafkaRequest } from '@rhoas/kafka-management
 import { useParams } from 'react-router-dom';
 import { useAuth, useConfig } from '@rhoas/app-services-ui-shared';
 
-export const getAdminServerUrl = (kafkaRequest?: KafkaRequest): string => {
+const DEFAULT_ADMIN_SERVER_URL_TEMPLATE = 'https://admin-server-{}/rest';
```
Comment (MikeEdgar) [inline src/app/pages/Kafka/kafka-instance.ts:6]: With recent SDK versions, the `/rest` should be removed.

**Comment by wtrocki** [comment]
@b1zzu Do you mind removing `/rest` from the endpoint. I think we going to get that change in stage soon

**Comment by wtrocki** [comment]
@b1zzu Do you mind removing `/rest` from the endpoint. I think we going to get that change in stage soon

File: src/app/pages/Kafka/kafka-instance.ts (hunk 1 above)
Comment (wtrocki) [inline src/app/pages/Kafka/kafka-instance.ts:6]:
```suggestion
const DEFAULT_ADMIN_SERVER_URL_TEMPLATE = 'https://admin-server-{}/';
```

**Pull request merged**